Wabbajack.App.Wpf/View Models/Installers/InstallerVM.cs

```diff
@@ -325,25 +380,41 @@ private IEnumerable<ErrorResponse> Validate()
         { 
             yield return ErrorResponse.Fail("Installing in this folder may overwrite Wabbajack");
         }
+        bool OnlyDownloadFolderPresent = false;
```

Comment:
Use PascalCase only on properties on the class itself, for variables it's standard to use camelCase.